### ISSUE TITLE
Custom intent broken since cargo-apk 0.9.0

### ISSUE THREAD

**Issue by Hoodad** [rust-windowing/android-ndk-rs#304]
Hello!

As the title suggests the custom intent filtration appears to been broken since release 0.9.0. In cargo-apk version 0.8.2 things worked as expected. And looking into what code changed between 0.8.2 and 0.9.0 I found this https://github.com/rust-windowing/android-ndk-rs/pull/241 though I'm not sure if this the culprit.

This issue can be reproduced by building using cargo-apk 0.8.2 and view what intents the app have registered to and building the same app using a newer of cargo-apk.

**Comment by MarijnS95** [comment]
Can you show the declaration of custom intent filters and the `aapt dump xmltree` output (or pretty-printed from `cargo-apk`) on both versions?

This was a breaking change iirc because of the rename from `intent_filter` to `intent_filters`.

**Comment by MarijnS95** [comment]
> as your PR addresses the root issue and fixes it

That means you've tested the PR? Thanks for confirming!

> Side-note, the `aapt dump xmltree` was a nice tool and I'm happy you introduced it :smile: 

There are more cool tools in there! Note that there's also `aapt2` as the successor to `aapt`!


Glad I could be of (quick) help, and also apologies for breaking it in the first place!

**Comment by Hoodad** [comment]
> That means you've tested the PR? Thanks for confirming!

No I just tested using 0.8.2 and 0.9.2 and can confirm that its the intent that are the difference. 

While I just now also tried verifying the PR by building locally I cannot seem to get configuration in line to use LLD rather then cc (issues related to me being on Windows). I tried setting up a `cargo\config.toml` specifying 

```toml
[target.aarch64-linux-android]
rustflags = ["-C", "linker-flavor=ld.lld"]
```

But no luck 😞 I'm stuck at `error: linker 'cc' not found`.
